source/clear-linux/get-started/bootable-usb/bootable-usb.rst

```diff
@@ -1,32 +1,203 @@
 
-.. _download-usb-image:
+* :ref:`bootable-usb-linux`
+* :ref:`bootable-usb-mac`
+* :ref:`bootable-usb-windows`
 
-Download the latest |CL| image
-******************************
+Prerequisites
+*************
 
-There are 2 types of |CL| images suitable for burning onto and running
-off a USB drive:
+* Use a **16GB** or larger USB drive.
+* `Download`_ the |CL| live boot image or interactive installer image.
 
-* Live image: :file:`clear-[version number]-live.img.xz`
-* Installer image: :file:`clear-[version number]-installer.img.xz`
+.. _bootable-usb-linux:
 
-Go to the |CL| `image`_ repository and download the desired type.
+Create a bootable USB drive on Linux
+************************************
 
-With the appropriate image downloaded, choose the step-by-step instructions
-applicable to your system:
+Make sure you have have completed all `Prerequisites`_.
 
-.. toctree::
-    :maxdepth: 1
+Before burning the image onto your USB drive,
+:ref:`verify and decompress your image <verify-linux>`.
 
-    bootable-usb-linux
-    bootable-usb-windows
-    bootable-usb-mac
+Burn the |CL| image onto a USB drive
+====================================
 
-.. _image: https://download.clearlinux.org/image
+.. caution::
+
+   |CAUTION-BACKUP-USB|
+
+#. Open a terminal emulator and get root privilege.
+
+   .. code-block:: bash
+
+      sudo -s
+
+#. Go to the directory with the decompressed image.
+#. Plug in the USB drive.
+#. Identify the USB drive using the :command:`lsblk` command. This shows all
+   drives attached to the system, including the primary hard disk. In the
+   example output below, there are 4 drives
+   (`/dev/sda`, `/dev/sdb`, `/dev/sdc`, and `/dev/sdd`) attached, where
+   `/dev/sda` is primary drive in this case. The remaining are 3 USB drives.
+   The output also shows the mounted partitions (under the `MOUNTPOINT`
+   column) for each drive.
+
+   .. code-block:: bash
+
+      lsblk
+
+   Example output:
+
+   .. code-block:: console
+
+      NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+      sdd      8:48   1    15G  0 disk
+      ├─sdd2   8:50   1     5G  0 part /run/media/user1/960c184f-3bb7-42b7-bcaf-0c1282
+      ├─sdd3   8:51   1     8G  0 part /run/media/user1/704f3382-b26d-4f34-af1b-cb9aab
+      └─sdd1   8:49   1     2G  0 part
+      sdb      8:16   1  14.8G  0 disk
+      └─sdb1   8:17   1  14.8G  0 part /run/media/user1/PATRIOT_USB
+      sdc      8:32   1   7.3G  0 disk
+      └─sdc1   8:33   1   7.3G  0 part /run/media/user1/LINUX MINT
+      sda      8:0    0 335.4G  0 disk
+      ├─sda4   8:4    0    28G  0 part
+      ├─sda2   8:2    0   3.7G  0 part [SWAP]
+      ├─sda7   8:7    0     6G  0 part /home
+      ├─sda5   8:5    0     1G  0 part /boot
+      ├─sda3   8:3    0   954M  0 part /boot/efi
+      ├─sda1   8:1    0    28G  0 part
+      ├─sda8   8:8    0    30G  0 part /
+      └─sda6   8:6    0   7.9G  0 part [SWAP]
+
+#. Before an image can be burned onto a USB drive, it should be un-mounted.
```

Comment:
Make this stronger + join to next sentence, something like: 
You must unmount a USB drive before you can burn it with an image, however, some Linux``\``* distros automatically... 
(Add escape character in Linux and delete "may")